R/drive_download.R

```diff
@@ -0,0 +1,72 @@
+#' Download a file from Google Drive.
+#'
+#' @template file
+#' @param out_path Character. Path for output file. Google files can
+#'   be downloaded to types specified [here](https://developers.google.com/drive/v3/web/manage-downloads).
```

Comment:
I'm weird about links but I try to avoid vague link text like "here". I try to make it clear where I'm sending people. So either use angle brackets and reveal the URL or come up with descriptive text.

BTW please call me out on same if you see it elsewhere.